### PR TITLE
new function to plot profile in 2D 

### PR DESCRIPTION
**Pull request by JuleRadtke**
#10 
grafik

### CODE REVIEW

File: eurec4a_environment/plot/profile.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ax.xaxis.set_major_formatter(myFmt)
+    fig.colorbar(the_fig, label=cbar_label)
+    ax.set_xlabel("time")
+    ax.set_ylabel("height / m")
+    ax.legend()
+    ax.xaxis.set_major_formatter(myFmt)
+    fig.autofmt_xdate()
+    ax.autoscale_view()
+    plt.tight_layout()
+    fig.savefig("ProfilePlot2D_{}.pdf".format(variable), bbox_inches="tight")
```
Comment (leifdenby) [inline eurec4a_environment/plot/profile.py:58]: I think I would leave it to the user to decide whether they want to save the figure. I would just return `ax` instead

Comment (JuleRadtke) [reply eurec4a_environment/plot/profile.py:58]: Okay, I change this!

Comment (annalea-albright) [reply eurec4a_environment/plot/profile.py:58]: Looks great, Jule!

**Comment by leifdenby** [comment]
This code is _so_ good 🌟 I love that you even say in the docstring where the kwargs are going

**Pull request merged**